configs/kinetics400/kinetics400_2d_rgb_r50_seg3.py

```diff
@@ -0,0 +1,129 @@
+model = dict(
+    type='TSN2D',
+    backbone=dict(
+        type='ResNet',
+        pretrained='modelzoo/resnet50-19c8e357.pth',
+        depth=50,
+        out_indices=(3,),
+        bn_eval=False,
+        partial_bn=False),
+    spatial_temporal_module=dict(
+        type='SimpleSpatialModule',
+        spatial_type='avg',
+        spatial_size=7),
+    segmental_consensus=dict(
+        type='SimpleConsensus',
+        consensus_type='avg'),
+    cls_head=dict(
+        type='ClsHead',
+        with_avg_pool=False,
+        temporal_feature_size=1,
+        spatial_feature_size=1,
+        dropout_ratio=0.4,
+        in_channels=2048,
+        num_classes=400))
+train_cfg = None
+test_cfg = None
+# dataset settings
+dataset_type = 'RawFramesDataset'
+data_root = '/mnt/SSD/Kinetics_extracted'
```

Comment:
please change this to `data/kinetics400/rawframes_train/` as in 

https://github.com/open-mmlab/mmaction/blob/master/configs/kinetics400/i3d_kinetics400_3d_rgb_r50_c3d_inflate3x1x1_seg1_f32s2.py#L40